modules/pytket-qulacs/pytket/extensions/qulacs/backends/qulacs_backend.py

```diff
@@ -76,17 +79,43 @@ class QulacsBackend(Backend):
     _supports_expectation = True
     _expectation_allows_nonhermitian = False
     _persistent_handles = False
+    _GATE_SET = {
+        OpType.X,
+        OpType.Y,
+        OpType.Z,
+        OpType.H,
+        OpType.S,
+        OpType.Sdg,
+        OpType.T,
+        OpType.Tdg,
+        OpType.Rx,
+        OpType.Ry,
+        OpType.Rz,
+        OpType.CX,
+        OpType.CZ,
+        OpType.SWAP,
+        OpType.U1,
+        OpType.U2,
+        OpType.U3,
+        OpType.Measure,
+        OpType.Barrier,
+    }
 
     def __init__(self) -> None:
         super().__init__()
+        name = "qulacs state simulator"
```

Comment:
Will these names be used programatically by mushroom? If so I think more care needs to be taken to come up with an agreed convention,